knowledge-base/dotnet-maui-editor-scroll-inside-scrollview.md

```diff
@@ -0,0 +1,118 @@
+---
+title: .NET MAUI Editor inside ScrollView
+description: Learn how to scroll the .NET MAUI Editor when in a ScrollView and scrolling the Editor's text when Editor is wrapped in a Border.
+type: how-to
+page_title: How to Scroll the Editor in a ScrollView - Editor for .NET MAUI
+slug: dotnet-maui-editor-scroll-inside-scrollview
+tags: editor, scrollview, scrolling, .NET MAUI, multiline entry
+res_type: kb
+---
+
+## Environment
+
+| Product |
+| --- |
+| Editor for .NET MAUI |
+
+## Description
+
+I want the <a href="https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/editor?view=net-maui-8.0" target="_blank">`Editor` for .NET MAUI</a> to scroll on Android when added inside a `ScrollView`.
+
+This KB article also answers the following questions:
+- How to use the .NET MAUI `Editor`, so the editor to scroll when having more input elements on the page?
```

Comment:
* Don't use the inline code style for the component names.
* Use a capital letter when mentioning the component's name (Editor).
*  Both comments apply to all occurrences of `Editor` and "editor" in this article.
```suggestion
How to use the .NET MAUI Editor, so the Editor scrolls when having more input elements on the page?
```